### ISSUE TITLE
Add TOC widget to each site chapter

### ISSUE THREAD

**Issue by bebraw** [survivejs/site#10]
Ie. similar to http://yeswejekyll.com/.

Migrated from https://github.com/survivejs/webpack_react/issues/50 .

**Comment by bebraw** [comment]
@Foxandxss Any thoughts on this? Do you think it would be handy to have a TOC within a chapter? Worth implementing?

**Comment by Foxandxss** [comment]
I don't follow. You mean the moving sidebar with the TOC?

**Comment by bebraw** [comment]
Yeah. It would get generated based on chapter contents and allow you to move between items fast (animated, updated browser hash).
